auditorium/stores/consent.js

```diff
@@ -3,28 +3,59 @@ var vault = require('offen/vault')
 module.exports = store
 
 function store (state, emitter) {
-  emitter.on('offen:consent', function (allow) {
+  emitter.on('offen:express-consent', function (status, callback) {
     vault(process.env.VAULT_HOST || '/vault/')
       .then(function (postMessage) {
         var consentRequest = {
-          type: 'CONSENT',
+          type: 'EXPRESS_CONSENT',
           payload: {
-            expressConsent: allow
+            status: status
           }
         }
         return postMessage(consentRequest)
       })
-      .then(function () {
-        if (allow) {
+      .then(function (message) {
+        state.consentStatus = message.payload
+        /*
+        if (message.payload.status === 'allow') {
           state.flash = __('Your have now opted in. Use the Auditorium to review and manage your data at any time.')
         } else {
-          state.flash = __('Your have now opted out and all usage data has been deleted.')
+          state.flash = __('Your have successfully opted out.')
```

Comment:
What's the rationale behind not keeping these? In case we really do not want them we can just get rid of the commented out portion I guess.

Reply:
we might have to rework the messaging because with this part in there is kind of a double - also we should reflect "opted out and there was data which is deleted now" and just "you are opted out"